### PR TITLE
Add getting started with Eclipse tutorial

### PR DESCRIPTION
**Pull request by goodwinnk**

### CODE REVIEW

File: docs/tutorials/getting-started-eclipse.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Getting Started with Eclipse Luna"
+description: "This tutorial guides through installing Kotlin plugin for Eclipse and creating Hello World application."
```
Comment (hhariri) [inline docs/tutorials/getting-started-eclipse.md]: guides *us* through installing *the* Kotlin plugin for Eclipse *and* create *a* Hello World application"

File: docs/tutorials/getting-started-eclipse.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+authors: Nikolay Krasko
+date: 2015-03-28
+showAuthorInfo: true
+---
+### Setting up the environment
+We will show how to install Kotlin plugin in Eclipse Luna SR2 (4.4.2) and create a simple Hello World application.
```
Comment (hhariri) [inline docs/tutorials/getting-started-eclipse.md]: Don't think we need to repeat description here. I'd just mention version of Eclipse required (if necessary)

File: docs/tutorials/getting-started-eclipse.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+1. Start from getting and installing [Eclipse Luna](https://www.eclipse.org/downloads/). "Eclipse IDE for Java Developers"
```
Comment (hhariri) [inline docs/tutorials/getting-started-eclipse.md]: Steps for other tutorials usually don't have numbers. Would be nice to use same style. Also the step name would be better with "Start from". It's already a step.

File: docs/tutorials/getting-started-eclipse.md (hunk 2 above)
Comment (hhariri) [inline docs/tutorials/getting-started-eclipse.md]: In fact I'd mention the Eclipse version in the first step

**Comment by hhariri** [comment]
There are a few grammar corrections needed. I can comment them inline (as I've done a couple) or it might just be easier if I merge and then correct. Let me know.

**Comment by goodwinnk** [comment]
Let's do a merging with the afterwards corrections. Completely agree that it will be more effective.

**Pull request merged**